edx_solutions_api_integration/courses/tests.py

```diff
@@ -785,27 +797,25 @@ def test_courses_overview_get_parsed(self):
         self.assertGreater(len(response.data), 0)
         self.assertIn(self.course.course_image, response.data['course_image_url'])
         sections = response.data['sections']
-        self.assertEqual(len(sections), 5)
+
+        self.assertEqual(len(sections), 4)
         self.assertIsNotNone(self._find_item_by_class(sections, 'about'))
         self.assertIsNotNone(self._find_item_by_class(sections, 'prerequisites'))
         self.assertIsNotNone(self._find_item_by_class(sections, 'course-staff'))
         self.assertIsNotNone(self._find_item_by_class(sections, 'faq'))
-        self.assertIsNotNone(self._find_item_by_class(sections, 'intro-video'))
 
         course_staff = self._find_item_by_class(sections, 'course-staff')
         staff = course_staff['articles']
-        self.assertEqual(len(staff), 3)
+
+        self.assertEqual(len(staff), 2)
         self.assertEqual(staff[0]['class'], "teacher")
         self.assertEqual(staff[0]['name'], "Staff Member #1")
-        self.assertEqual(staff[0]['image_src'], "/images/pl-faculty.png")
+        self.assertIn("images_placeholder-faculty.png", staff[0]['image_src'])
         self.assertIn("<p>Biography of instructor/staff member #1</p>", staff[0]['bio'])
         self.assertEqual(staff[1]['class'], "teacher")
         self.assertEqual(staff[1]['name'], "Staff Member #2")
-        self.assertEqual(staff[1]['image_src'], "/images/pl-faculty.png")
+        self.assertIn("images_placeholder-faculty.png", staff[1]['image_src'])
         self.assertIn("<p>Biography of instructor/staff member #2</p>", staff[1]['bio'])
-        self.assertEqual(staff[2]['class'], "author")
```

Comment:
what is the reason for removing this and next assertion?

Reply:
We are now getting only two staff members, and both are teachers.